usecases/scheduled_execution/run_scheduled_execution.go

```diff
@@ -198,10 +198,6 @@ func (usecase *RunScheduledExecution) insertAsyncDecisionTasks(
 		return err
 	}
 
-	if len(objectIds) == 0 {
```

Comment:
Turns out this was too smart: early exit skipped the insertion of the job that does the batch exec status handling, at no gain (because `len()==0` is not really a special case anyway)